### ISSUE TITLE
Attribute Values pairs longest list rule issue

### ISSUE THREAD

**Issue by mnaugle-tt** [tt-acm/Spectacles.GrasshopperExporter#22]
Related to #21 

Attribute Value pairs should work this way too:
![image](https://cloud.githubusercontent.com/assets/3228216/11305789/4783b352-8f7c-11e5-9c17-3378804c23d0.png)

**Comment by bhowes-tt** [comment]
As I mentioned in #23 ,  this feels like it's asking for trouble (or at least a lack of clear meaning) downstream. but if it is being requested in the interests of a less error prone UX, then ok, i buy it.

**Comment by mnaugle-tt** [comment]
@bhowes-tt if you don't like this or #23 then how do I create an attribute name called Z height and have a whole bunch of mesh faces have their own unique value?  Right now I have to make a unique Key for each value.

**Comment by bhowes-tt** [comment]
@mnaugle-tt , assuming you want to select a face and see it's Z height attribute in the browser, each face should be it's own mesh.  

If we laced it up the way you are asking, you would select the one mesh in the browser, and get a list of 'Z height' attributes in the element attributes dialog - one per face.  Is that what you want?

**Comment by mnaugle-tt** [comment]
I would expect this to work:
![image](https://cloud.githubusercontent.com/assets/3228216/11312475/44bacbec-8fa4-11e5-9840-ed4472de6a96.png)

Each face of the mesh is an individual item in a list, there is a single value going to [aN] and list of values that matches the length of the M items going to [aV].  

So my next guess was to try this:
![image](https://cloud.githubusercontent.com/assets/3228216/11312477/49efe174-8fa4-11e5-8db7-5aea295848a6.png)

only being able to get there by grafting and using a repeat data component:
![image](https://cloud.githubusercontent.com/assets/3228216/11312517/859c6cd8-8fa4-11e5-9219-4b3f6a3009e6.png)

The first option is what I would expect as the best / right way to do this.

**Comment by mnaugle-tt** [comment]
apologies for what i now see as misleading initial example.  i had removed all my other plugins when testing this morning so didn't have the ability explode that mesh.  if you look at the this set of three code blocks hopefully that clarifies the issue better.

**Comment by bhowes-tt** [comment]
ok - that last one is almost there.  replace the aN input with the single 'z height' panel and you'll be there i think.

**Comment by mnaugle-tt** [comment]
yes that works, but why not the first?  the first is what i would consider 'standard' grasshopper logic.
![image](https://cloud.githubusercontent.com/assets/3228216/11313509/56c66eca-8fab-11e5-81f2-5e658da57a3a.png)

**Comment by mnaugle-tt** [comment]
I see where you are coming from with this, fine to leave it as is.  Just doesn't feel intuitive to have to either graph or repeat data especially in the instance of the first example (well actually the 2nd screen shot).

**Comment by anagpuyol** [comment]
I think this becomes more intuitive when you want to write more data into each mesh. Then, the way the data is written seems clean. Leaving it as is then
